tools/src/main/scala/scala/scalanative/optimizer/Pass.scala

```diff
@@ -135,6 +135,11 @@ trait Pass {
     case Op.Select(v1, v2, v3) =>
       Op.Select(txVal(v1), txVal(v2), txVal(v3))
 
+    case Op.Box(code, obj) =>
+      Op.Box(code, txVal(obj))
+    case Op.Unbox(code, obj) =>
+      Op.Unbox(code, txVal(obj))
+
```

Comment:
Similarly to case class definitions for ops we group cases into low-level and high-level. Those two should go in the end of this match.